spec/models/user_spec.rb

```
@@ -0,0 +1,69 @@
+#  age        :string(255)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user) do
+    build(
+      :user,
+      first_name:,
```

RuboCop (1.69.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)